test/integration/transaction.spec.ts

```diff
@@ -525,6 +525,72 @@ describe('liquidjs-lib (transactions with psbt)', () => {
     // to build and broadcast to the actual Bitcoin network, see https://github.com/bitcoinjs/bitcoinjs-lib/issues/839
   });
 
+  it(
+    'can create (and broadcast via 3PBP) a confidential Transaction' +
+      ' with confidential AND unconfidential outputs',
+    async () => {
+      // these are { payment: Payment; keys: ECPair[] }
+      const alice0 = createPayment('p2pkh', undefined, undefined, true); // confidential
+      const bob0 = createPayment('p2pkh', undefined, undefined, false); // unconfidential
+
+      const aliceBlindingPubKey = liquid.ECPair.fromPrivateKey(
+        alice0.blindingKeys[0],
+      ).publicKey!;
+
+      // give Alice unspent outputs
+      const inputData1 = await getInputData(alice0.payment, false, 'noredeem');
+      {
+        const {
+          hash, // string of txid or Buffer of tx hash. (txid and hash are reverse order)
+          index, // the output index of the txo you are spending
+          nonWitnessUtxo, // the full previous transaction as a Buffer
+        } = inputData1;
+        assert.deepStrictEqual({ hash, index, nonWitnessUtxo }, inputData1);
+      }
+
+      // network is only needed if you pass an address to addOutput
+      // using script (Buffer of scriptPubkey) instead will avoid needed network.
+      const psbt = new liquid.Psbt({ network: regtest })
+        .addInput(inputData1) // alice1 unspent
+        .addOutput({
+          asset,
+          nonce,
+          script: bob0.payment.output,
+          value: satoshiToConfidentialValue(5000_0000),
```

Comment:
please remove the `_` to keep code style consistency.
```suggestion
          value: satoshiToConfidentialValue(50000000),
```